src/store/transactions/withdraw/withdraw.actions.ts

```diff
@@ -0,0 +1,235 @@
+import { BigNumber } from "@ethersproject/bignumber";
+
+// domain
+import { EstimatedWithdrawFee } from "src/domain";
+import { Account, Exit, PoolTransaction } from "src/domain/hermez";
+
+export enum WithdrawActionTypes {
+  LOAD_ACCOUNT = "[WITHDRAW] LOAD ACCOUNT",
+  LOAD_ACCOUNT_SUCCESS = "[WITHDRAW] LOAD ACCOUNT SUCCESS",
+  LOAD_ACCOUNT_FAILURE = "[WITHDRAW] LOAD ACCOUNT FAILURE",
+  LOAD_EXIT = "[WITHDRAW] LOAD EXIT",
+  LOAD_EXIT_SUCCESS = "[WITHDRAW] LOAD EXIT SUCCESS",
+  LOAD_EXIT_FAILURE = "[WITHDRAW] LOAD EXIT FAILURE",
+  LOAD_POOL_TRANSACTIONS = "[WITHDRAW] LOAD POOL TRANSACTIONS",
+  LOAD_POOL_TRANSACTIONS_SUCCESS = "[WITHDRAW] LOAD POOL TRANSACTIONS SUCCESS",
+  LOAD_POOL_TRANSACTIONS_FAILURE = "[WITHDRAW] LOAD POOL TRANSACTIONS FAILURE",
+  LOAD_ESTIMATED_WITHDRAW_FEE = "[WITHDRAW] LOAD ESTIMATED WITHDRAW FEE",
+  LOAD_ESTIMATED_WITHDRAW_FEE_SUCCESS = "[WITHDRAW] LOAD ESTIMATED WITHDRAW FEE SUCCESS",
+  LOAD_ESTIMATED_WITHDRAW_FEE_FAILURE = "[WITHDRAW] LOAD ESTIMATED WITHDRAW FEE FAILURE",
+  START_TRANSACTION_APPROVAL = "[WITHDRAW] START TRANSACTION APPROVAL",
+  STOP_TRANSACTION_APPROVAL = "[WITHDRAW] STOP TRANSACTION APPROVAL",
+  RESET_STATE = "[WITHDRAW] RESET STATE",
+}
+
+export type Step = "load-data" | "build-transaction" | "review-transaction";
```

Comment:
Because we are not loading only the `account`, we are also loading the `poolTransactions`, so maybe it makes more sense to rename the other steps to `load-data` instead, as I think that most of them also load both things.

Reply:
I've removed the `build-transaciton` step as it's not used yep!

Reply:
> Because we are not loading only

Right, ok.